### DISCUSSION TITLE
Tag

### DISCUSSION THREAD

**Discussion by And96** [standardnotes/mobile#543] [Q&A]
How to find notes without tag? 

Is possible to show tags in the list of notes?

**Comment by atmoio** [comment]
https://standardnotes.com/help/42/how-do-i-view-a-list-of-untagged-notes-and-create-other-dynamic-filters

**Comment by And96** [comment]
I dont have Folders extension

**Comment by stefan-yas** [comment]
@And96 Just click on the "+" where the Folders are, as shown in the screenshot below, and add your dynamic filter there

![image](https://user-images.githubusercontent.com/52251483/153321335-ecd8d17e-7bdc-4d91-a433-5dc462f7d004.png)

**Reply by atmoio** [reply]
Note that smart filters are a paid feature. The screenshot above is from the 3.10 beta (which will be released next week).
